derive/src/utils.rs

```diff
@@ -37,6 +37,19 @@ where
 		attr.path().is_ident("codec").then(|| pred(attr.parse_args().ok()?)).flatten()
 	})
 }
+fn find_meta_item_in_path<F, R>(attrs: &[Attribute], word: &str, success_callback: F) -> Option<R>
```

Comment:
Personally I would name it `find_path_meta_item`.
I am not sure what `in_path` means in this context
```suggestion

/// Execute the callback for all the attribute matching `#[codec($path)]`.
fn find_path_meta_item<F, R>(attrs: &[Attribute], path: &str, success_callback: F) -> Option<R>
```